README.md

```diff
@@ -1 +1,58 @@
-# permawebjs
\ No newline at end of file
+# permawebjs
+
+## Introduction
+
+PermawebJS aims to lower the barrier of onboarding and building on Arweave by creating a well documented one-stop library.
+
+## Installation
+
+Run the following command to install the package
+
+```
+npm install permawebjs
+```
+
+### Using Yarn
+
+```
+yarn add permawebjs
```

Comment:
This can be combined with the npm install section itself with a `#OR` comment in the middle. Have seen most docs  use a similar format.

Reply:
good catch